templates/jet/app/navigation/NavigationWrapper.tsx

```diff
@@ -1,8 +0,0 @@
-import React from 'react';
```

Comment:
I think we still need this file. Exposing a ref from the `NavigationContainer` is pretty common use-case, and I personally don't like that logic muddying up the App.tsx file.

Reply:
Granted, App.tsx would need some minor refactoring in this case.

Reply:
@makozlo Did you see these comments? Just checking